internal/engine/buildcontroller.go

```diff
@@ -220,9 +220,9 @@ func (c *BuildController) logBuildEntry(ctx context.Context, entry buildEntry, c
 
 	l := logger.Get(ctx)
 	if firstBuild {
-		p := logger.Blue(l).Sprintf("\n──┤ Building: ")
+		p := logger.Blue(l).Sprintf("──┤ Building: ")
```

Comment:
There's a bug in this PR (and also the existing PrefixedWriter) - if something logs a colored newline, that color will also affect that next line's timestamp/prefix.

This is annoying enough to fix and low enough impact (colored prefixes!) that I figured band-aiding it for now is fine.